pkg/sti/script/installer.go

```diff
@@ -108,6 +109,20 @@ func (s *handler) download(scripts []api.Script, workingDir string) (bool, error
 		}
 	}
 
+	// check for the scripts inside the sources
+	scriptsDir := filepath.Join(workingDir, "/upload/src/.sti/bin")
+	if s.fs.Exists(scriptsDir) {
+		for _, script := range scripts {
+			file := filepath.Join(scriptsDir, string(script))
+			if s.fs.Exists(file) {
+				downloads[script] <- true
+			} else {
+				errs[script] <- errors.NewScriptDownloadError(script, nil)
```

Comment:
it doesn't look like the downloadAsync code reports an error for missing scripts, it just returns if !download

if "download" can be true when the download failed because the file didn't exist, then this code needs commenting.